Added RN for fix MTM-56734 (#1847)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-10.20.266.0-Root-level-managed-object-is-not-obtained-on-query.md

---
date: ""

> title: Root level managed object is not found by querying with the onlyRoots flag

Reviewer: ```suggestion
title: Root level managed object is now reliably found when querying with the onlyRoots flag
```

Author: Changed.

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-OG_650_b2
    label: Core platform
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-56734
version: 10.20.266.0
---

> Fixed an issue in inventory with finding an asset by the onlyRoots flag when the object is removed from the asset hierarchy and becomes the root object again.

Reviewer: ```suggestion
In the inventory, when querying with the `onlyRoots` flag, assets could not be found if the object was removed from the asset hierarchy and became the root object again. This issue has now been fixed and root level managed objects are found when querying with the `onlyRoots` flag.
```

Author: Changed.
